pkg/apis/org/v1/che_types.go

```diff
@@ -81,6 +81,8 @@ type CheClusterSpecServer struct {
 	// By default a dedicated plugin registry server is started.
 	// But if ExternalPluginRegistry is `true`, then no such dedicated server will be started by the operator
 	ExternalPluginRegistry bool `json:"externalPluginRegistry"`
+	// OverrideCheProperties is a list of EnvVars of Che properties to override from their default value
```

Comment:
As a comment, I would put something like:

OverrideCheProperties is a list of additional environment variables that will be applied in the che config map, in addition to the values already generated from other fields of the custom resource (CR).
If OverrideCheProperties contains a property that would be normally generated in che config map from other CR fields, then the value in the OverrideCheProperties will be used.